### PR TITLE
Chore: Migrate to create plugin

### PR DESCRIPTION
**Pull request by tolzhabayev**
Migrating this panel to use create-plugin instead of toolkit most of the files are auto generated by the migration tool which worked extremely well.

**Additional changes:**
- Bump version to `2.1.1`
- Update packages / dependencies
- Introduce tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,34 @@
-node_modules
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (jackw) [inline .gitignore:6]: Hmm... the `yarn-error.log` appears to have crept into this PR. 🤔

File: src/ClockPanel.test.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
+import { ClockPanel } from "ClockPanel";
+import {render} from '@testing-library/react'
+import '@testing-library/jest-dom'
```
Comment (jackw) [inline src/ClockPanel.test.tsx:3]: Hmmm... you shouldn't need to do this if the [jest-setup](https://github.com/grafana/clock-panel/pull/110/files#diff-1c29b76856438cc695cc32aa47ea0b678d925da633b749ef1b5fd90b882951a0R7) is working correctly.

File: src/ClockPanel.test.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import { ClockPanel } from "ClockPanel";
```
Comment (jackw) [inline src/ClockPanel.test.tsx:1]: So great you've added some tests for this plugin! 👏👏👏

File: package.json [hunk 4]
```diff
@@ -18,20 +25,59 @@
...
+    "webpack-livereload-plugin": "^3.0.2"
   },
   "dependencies": {
-    "moment-timezone": "0.5.34"
-  },
-  "volta": {
-    "node": "14.17.6",
-    "yarn": "1.22.11"
+    "@emotion/css": "^11.1.3",
+    "@grafana/data": "9.1.2",
```
Comment (jackw) [inline package.json:76]: Did you bump these `@grafana` dependencies or did the create-plugin migrate script do it?

Comment (tolzhabayev) [reply package.json:76]: ~Gangster rap made me do it~ `create-plugin migrate` script or the `create-plugin update` script did it, don't remember exactly.

File: .gitignore (hunk 1 above)
Comment (tolzhabayev) [reply .gitignore:6]: Oh, right - removed https://github.com/grafana/clock-panel/pull/110/commits/2b39f0a664cb97cbff0c16c0d87a3597b7eab641

File: src/ClockPanel.test.tsx (hunk 2 above)
Comment (tolzhabayev) [reply src/ClockPanel.test.tsx]: Yep, I should've looked into what jest setup is doing :) 

Removed - https://github.com/grafana/clock-panel/pull/110/commits/2b39f0a664cb97cbff0c16c0d87a3597b7eab641

File: package.json (hunk 4 above)
Comment (jackw) [reply package.json:76]: Hmm... gonna need to take a look at that then. I don't think we should ever tamper with a plugin grafana dependencies unless it's a new scaffold.

**Review by jackw — APPROVED** [review]
LGTM! 🚀 

Looks like you need to sign the CLA? 🤔

**Comment by tolzhabayev** [comment]
CLA signed :) I will merge and promote this next week

**Review by mckn — APPROVED** [review]
LGTM! Note: I have not tested the plugin only done a review of the code.

**Pull request merged**